common/include/common/config/GameConfig.h

```diff
@@ -52,6 +52,13 @@ struct GameConfig
     CfgVar<bool> nearest_texture_filtering = false;
     CfgVar<unsigned> msaa = 0;
 
+    enum ClampMode
+    {
+        AUTOMATIC,
+        CLASSIC,
+        FULL,
+    };
```

Comment:
Scoped enumerations do not implicitly cast and clash less with names. For variant names snake case is used by rafalh where pascal case is expected.
```suggestion
    enum class ClampMode {
        automatic,
        classic,
        full,
    };
```